src/result/__init__.py

```diff
@@ -8,4 +8,4 @@
     "UnwrapError",
     "as_result",
 ]
-__version__ = "0.8.0"
+__version__ = "0.0.0"
```

Comment:
a PEP 440 compliant development version is a better choice (see 
https://peps.python.org/pep-0440/#developmental-releases)

```suggestion
__version__ = "0.9.0.dev0"
```


see for example: https://github.com/python-attrs/attrs/blob/main/src/attr/__init__.py#L35

Reply:
Thanks